### PR TITLE
Fix Facebook Connector by adapting it to the new message format

### PR DESCRIPTION
**Pull request by Neitsch**
Hi,

I just wrote a small blog post about this project and my friend wanted to try it out. Turns out Facebook changed the format of the data. This should fix it.

### CODE REVIEW

File: facebook_connector.py [hunk 1]
```diff
@@ -10,15 +10,15 @@
 from __future__ import print_function
 
 from bs4 import BeautifulSoup
-from fbchat_archive_parser.parser import MessageHtmlParser
+from fbchat_archive_parser.parser import parse
```
Comment (mdezube) [inline facebook_connector.py]: When there's a risk of conflict it's better to write:

from fbchat_archive_parser import parser then below parser.parse()

File: facebook_connector.py [hunk 2]
```diff
@@ -126,7 +120,9 @@ def get_cleaned_fully_merged_messages(strip_html_content=True,
     if not _messages_file:
         print("Please initialize the facebook_connector module.")
         return
-    chats = MessageHtmlParser(path=_messages_file).parse()
+    chats = None
+    with io.open(_messages_file, mode="rt", encoding="utf-8") as handle:
```
Comment (mdezube) [inline facebook_connector.py:124]: Why this instead of just with open()?  I don't usually see it fully written out.

Also what's mode "rt" instead of "r"?

**Review by mdezube — COMMENTED** [review]
Awesome, thanks for fixing this!  Also where's the blog post?  I'd love to read it.

File: facebook_connector.py (hunk 2 above)
Comment (Neitsch) [reply facebook_connector.py:124]: io.open differs from open. It allows specifying the encoding. I tried open, but it just produced garbage. 'rt' is 'read' and 'text mode'.

**Comment by Neitsch** [comment]
I just started the blog, so my writing is still really rough, but here's the post :)
https://www.nigel-schuster.de/articles/messaging-analysis-and-visualization

**Comment by mdezube** [comment]
Thanks for the PR!

**Comment by mdezube** [comment]
Also very excited to read the blog post, will do so Monday.

**Pull request merged**